e2e/src/fullstack.js

```diff
@@ -1241,25 +1244,35 @@ function newPRuntime(teePort, tmpPath, name = 'app') {
     const workDir = path.resolve(`${tmpPath}/${name}`);
     const sealDir = path.resolve(`${workDir}/data`);
     if (!fs.existsSync(workDir)) {
-        fs.mkdirSync(workDir);
-        fs.mkdirSync(sealDir);
-        const filesMustCopy = ['Rocket.toml', pRuntimeBin];
-        const filesShouldCopy = ['GeoLite2-City.mmdb']
-        filesMustCopy.forEach(f =>
-            fs.copyFileSync(`${path.dirname(pathPRuntime)}/${f}`, `${workDir}/${f}`)
-        );
-        filesShouldCopy.forEach(f => {
-            if (fs.existsSync(`${path.dirname(pathPRuntime)}/${f}`)) {
-                fs.copyFileSync(`${path.dirname(pathPRuntime)}/${f}`, `${workDir}/${f}`)
-            }
-        });
+        if (inSgx) {
+            fs.cpSync(pRuntimeDir, workDir, { recursive: true })
+            fs.mkdirSync(path.resolve(`${sealDir}/protected_files/`), { recursive: true });
+            fs.mkdirSync(path.resolve(`${sealDir}/storage_files/`), { recursive: true });
+        } else {
+            fs.mkdirSync(sealDir, { recursive: true });
+            const filesMustCopy = ['Rocket.toml', pRuntimeBin];
+            const filesShouldCopy = ['GeoLite2-City.mmdb']
```

Comment:
Let's file an issue to remove GeoLite access completely. We are going to use proactive network prob to replace geolocation.

Reply:
Let me just do it now.

Reply:
https://github.com/Phala-Network/phala-blockchain/pull/995